src/PrestaShopBundle/Resources/config/services/adapter/product/_combination.yml

```diff
@@ -168,10 +168,18 @@ services:
             - name: tactician.handler
               command: PrestaShop\PrestaShop\Core\Domain\Product\Combination\Command\RemoveAllCombinationImagesCommand
 
-    prestashop.adapter.product.combination.command_handler.remove_combination_handler:
```

Comment:
It's a BC Break as this service has been defined in 178x. Could you document it ?

Reply:
:+1: updated in the PR description